cdc/sinkv2/ddlsink/factory/factory.go

```diff
@@ -38,13 +40,14 @@ func New(
 		return nil, err
 	}
 	schema := strings.ToLower(sinkURI.Scheme)
-	// TODO: add more sink factory here.
 	switch schema {
 	case sink.KafkaSchema, sink.KafkaSSLSchema:
 		return mq.NewKafkaDDLSink(ctx, sinkURI, cfg,
 			kafka.NewAdminClientImpl, ddlproducer.NewKafkaDDLProducer)
 	case sink.BlackHoleSchema:
 		return blackhole.New(), nil
+	case sink.MySQLSSLSchema, sink.MySQLSchema, sink.TiDBSchema, sink.TiDBSSLSchema:
+		return mysql.NewMySQLDDLSink(ctx, sinkURI, cfg, pmysql.CreateMySQLDBConn)
```

Comment:
How about remove parameter `pmysql.CreateMySQLDBConn`?

Reply:
I think this is acceptable because it is the internal method. And kafka does the same thing. Creating a new constructor would allow us to maintain the exact same code every time.

Reply:
The ability to walk the real code path during testing is also a major advantage of this approach.

Reply:
We got tons of config processes in the constructor, so if we can walk these codes it would be better. And we only need to maintain one copy of the code.